CONTRIBUTING.md

```diff
@@ -193,8 +193,8 @@ sbt assembly
 $SPARK_HOME/bin/spark-shell \
 --jars ./target/scala-2.12/qbeast-spark-assembly-0.6.0.jar \
```

Comment:
here we should also change the version of assembly to 0.8.0-SNAPSHOT

Reply:
In fact, I would suggest to update the whole example with this new version, so we can start developing from Refactoring